### DISCUSSION TITLE
Command palette: add compare

### DISCUSSION THREAD

**Discussion by johnkors** [community/community#7720] [Other Feature Feedback, Questions, & Ideas]
Hi!

I use the compare url quite a bit to be able to do a review over what will become the latest release before deployment:
1) Get the latest release/tag
2) Open the compare URL: main vs latest tag

Would be nice if there was a quick(er) more user friendly way to do it, other[ than just modifying the URL (source: github docs)](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/viewing-and-comparing-commits/comparing-commits)

> To compare different versions of your repository, append /compare to your repository's path.

For example, if the command palette had a compare command in some fashion (and/or if there was an actually link somewhere in the UI to do it). 

![image](https://user-images.githubusercontent.com/206726/141754856-dc38e7f7-b72a-48fc-8f56-42b98249713b.png)

**Comment by jpmcd** [comment]
I constantly wonder about this too. Pretty much every time I want to compare two branches or different commits on the same branch. Then I have to look up how to modify the URL while thinking "Why isn't there an easier way to do this?"
